pkg/cover/html.go

```diff
@@ -434,7 +434,7 @@ func groupCoverByFilePrefixes(datas []fileStats, subsystems []mgrconfig.Subsyste
 	return d
 }
 
-func (rg *ReportGenerator) DoHTMLTable(w io.Writer, progs []Prog, coverFilter map[uint32]uint32) error {
+func (rg *ReportGenerator) DoSubsystemCover(w io.Writer, progs []Prog, coverFilter map[uint32]uint32) error {
```

Comment:
(Not a change request and definitely in scope of this PR, just FYI)

I see you added the `mgrconfig.Subsystem` a couple of years ago. Now we also have a more advanced support of kernel subsystems (https://github.com/google/syzkaller/tree/master/pkg/subsystem), so it might make sense to integrate it with coverage. At least we could switch to `pkg/subsystem`'s data structures: https://github.com/google/syzkaller/blob/master/pkg/subsystem/entities.go and try to make use of pre-generated subsystems lists for Linux (https://github.com/google/syzkaller/blob/master/pkg/subsystem/lists/linux.go) to see how it goes.

Reply:
not related to this PR, so removed